noether_examples/src/thickness_simulator_client.cpp

```diff
@@ -0,0 +1,182 @@
+#include <ros/ros.h>
```

Comment:
To me this looks like a piece of code for testing. At least put in a comment block that describes what it does and how to use it.